multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/model/ImageTransformerImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.mikepenz.markdown.model
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.painter.Painter
+import com.mikepenz.markdown.utils.imagePainter
+import com.mikepenz.markdown.utils.painterIntrinsicSize
+
+internal object ImageTransformerImpl : ImageTransformer {
```

Comment:
I believe it would probably better to make this a normal class instead of an object. As we won't need it to exist if somebody provides their own implementation.